src/tests/multihost/adsites/test_adsites.py

```diff
@@ -0,0 +1,262 @@
+from __future__ import print_function
+import time
+import pytest
+from sssd.testlib.common.utils import sssdTools
+
+
+@pytest.mark.adsites
+class Testadsites(object):
+    """
+    @Title: IDM-SSSD-TC: ad_provider: adsites:
+    Improve AD site discovery process
+    Test cases for BZ: 1819012
+
+    @Steps:
+    1. Join client to AD
+    2. Start SSSD and enable debug
+    3. Create secondary site, move second domain controller to second site 
+    """
+    @pytest.mark.adsites
+    def test_001_ad_startup_discovery(self, multihost, adjoin):
+        """
+        @Title: IDM-SSSD-TC: ad_startup_discovery
```

Comment:
Couple questions about the docstrings format in use:
1.  Is it necessary to use @ to start keywords in the docstring?  I thought betelgeuse switched to : a while back?
2.  Are the bullet points after the title intended to be steps?  If so in betelgeuse, I think you need both steps and expected results and to be labeled with @steps or @expectedresults and the items to be indented I think.  Need to confirm that though.

If this is just a general documentation formatting, close this comment.